docroot/themes/humsci/humsci_basic/src/scss/utilities/_font-awesome.scss

```diff
@@ -18,8 +18,18 @@
   transition: 125ms color ease-out;
   @include hb-pairing-color('color', 'secondary');
 
+  &:hover,
```

Comment:
hmm will this apply to all elements that font awesome is on, not just links?

Reply:
I think it is

Reply:
Oh man! You're right. I need to revisit this then. Good catch!